extensions/amp-access/0.1/access-expr-impl.js

```diff
@@ -175,7 +175,13 @@ parseError: function parseError(str, hash) {
     if (hash.recoverable) {
         this.trace(str);
     } else {
-        throw new Error(str);
+        function _parseError (msg, hash) {
+            this.message = msg;
+            this.hash = hash;
+        }
+        _parseError.prototype = Error;
+
+        throw new _parseError(str, hash);
```

Comment:
specifically its this line https://github.com/ampproject/amphtml/pull/9775/files#diff-8d20dae7ae84978e5dba004cfa21ff72L304